Content.Client/UserInterface/Systems/Scoreboard/ScoreboardUIController.cs

```diff
@@ -0,0 +1,64 @@
+using Content.Client.RoundEnd;
+using Robust.Client.Input;
+using Content.Shared.Input;
+using Robust.Shared.Input.Binding;
+using Content.Client.GameTicking.Managers;
+using Content.Client.Lobby;
+using Content.Client.Gameplay;
+using Robust.Shared.Log;
+using Robust.Client.UserInterface;
+using Robust.Client.UserInterface.Controllers;
+
+namespace Content.Client.UserInterface.Scoreboard;
+
+public sealed class ScoreboardUIController : UIController, IOnStateEntered<LobbyState>, IOnStateEntered<GameplayState>, IOnStateExited<LobbyState>, IOnStateExited<GameplayState>
+{
+
+    [Dependency] private readonly IInputManager _input = default!;
+    [Dependency] private readonly IEntitySystemManager _entitySystem = default!;
+    private ClientGameTicker _gameTicker = default!;
```

Comment:
```suggestion
    [UISystemDependency] private readonly ClientGameTicker _gameTicker = default!;
```
Sloth earlier was talking about using a UISystemDependency, which is for using classes that inherit from EntitySystem to be injected as a dependency in UI classes.

In non-UI you would just use [Dependency] as normal for EntitySystems.